GiwazipClient/Views/PostingPhotoViewController.swift

```diff
@@ -24,77 +46,171 @@ class PostingPhotoViewController: BaseViewController {
         $0.numberOfLines = 2
         return $0
     }(UILabel())
-    
+
     private let photoCollectionView: UICollectionView = {
         let collectionView = UICollectionView(frame: .zero,
                                               collectionViewLayout: UICollectionViewFlowLayout())
+        collectionView.contentInset.bottom = 80
         return collectionView
     }()
-    
-    private let nextButton: UIButton = {
-        $0.setTitle("다음", for: .normal)
-        $0.setTitleColor(.white, for: .normal)
-        $0.backgroundColor = .blue
-        $0.layer.cornerRadius = 16
+
+    private lazy var nextButton: UIButton = {
+        $0.configuration?.title = "다음"
+        $0.configuration?.attributedTitle?.font = UIFont.systemFont(ofSize: 18, weight: .semibold)
+        $0.configuration?.baseForegroundColor = .white
+        $0.configuration?.contentInsets.bottom = 20
+        $0.backgroundColor = .gray
+        $0.addTarget(self, action: #selector(showAlert), for: .touchUpInside)
         return $0
-    }(UIButton())
-    
+    }(UIButton(configuration: uiButtonConfiguration))
+
     // MARK: - Method
-    
+
     override func attribute() {
         super.attribute()
-        
+
+        setupCollectionView()
+    }
+
+    private func setupCollectionView() {
         photoCollectionView.delegate = self
         photoCollectionView.dataSource = self
-        photoCollectionView.register(PostingPhotoCell.self, forCellWithReuseIdentifier: PostingPhotoCell.identifier)
+        photoCollectionView.register(PostingPhotoCell.self,
+                                     forCellWithReuseIdentifier: PostingPhotoCell.identifier)
     }
     
     override func layout() {
         super.layout()
-        
+
         view.addSubview(guidanceLabel)
         guidanceLabel.snp.makeConstraints {
             $0.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(12)
```

Comment:
p3;
view.safeAreaLayoutGuide.snp.top 까지 붙이신 특별한 이유가 있으실까요?
없다면 그냥 항상 쓰던 것처럼 쓰는 게, 일관성 있지 않을까?? 라는 생각이 들어서 여쭤봅니다! : )

```suggestion
        guidanceLabel.snp.makeConstraints {
            $0.top.equalTo(view.safeAreaLayoutGuide).offset(12)
```

Reply:
> p3; view.safeAreaLayoutGuide.snp.top 까지 붙이신 특별한 이유가 있으실까요? 없다면 그냥 항상 쓰던 것처럼 쓰는 게, 일관성 있지 않을까?? 라는 생각이 들어서 여쭤봅니다! : )

3a14d7b
저는 .snp.top 등을 작성하는게 더 명확하고 자세한 코드가 아닐까 라고 생각했는데, 제거해도 해석이 모호해지지 않는 것 같네요 :)